### PR TITLE
CSS [Edit] CSS animation-direction

### PR DESCRIPTION
**Pull request by Kadif-Ravik**
### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/animations/terms/animation-direction/animation-direction.md [hunk 1]
```diff
@@ -84,3 +84,6 @@ div {
   }
 }
 ```
+![Example 1 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-direction-example1.gif)
+
+![Example 2 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-direction-example2.gif)
```
Comment (dakshdeepHERE) [inline content/css/concepts/animations/terms/animation-direction/animation-direction.md:89]: Can we give better names to these tags? As if this gif doesn't load then this text will show.

**Review by dakshdeepHERE — COMMENTED** [review]
I only have one question for you @Kadif-Ravik. Let me know what you think of it.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review

File: content/css/concepts/animations/terms/animation-direction/animation-direction.md [hunk 2]
```diff
@@ -51,6 +51,8 @@ div {
 }
 ```
 
+![Animation of example 1 with animation direction as reverse](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-direction-example1.gif)
```
Comment (ishg-153) [inline content/css/concepts/animations/terms/animation-direction/animation-direction.md:54]:
```suggestion
![Animation with reverse animation-direction](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-direction-example1.gif)
```

File: content/css/concepts/animations/terms/animation-direction/animation-direction.md [hunk 3]
```diff
@@ -84,3 +86,5 @@ div {
   }
 }
 ```
+
+![Animation of example 2 with animation direction as alternate and altrernate-reverse](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-direction-example2.gif)
```
Comment (ishg-153) [inline content/css/concepts/animations/terms/animation-direction/animation-direction.md:90]:
```suggestion
![Animation with animation-direction as alternate and altrernate-reverse](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-direction-example2.gif)
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Kadif-Ravik I've suggested minor changes in the `alt-text` (text in square brackets) of the output images. Please make those.

**Review by ishg-153 — APPROVED** [review]
@Kadif-Ravik This is ready to merge.

**Pull request merged**